styx-service-common/src/main/java/com/spotify/styx/state/TimeoutConfig.java

```diff
@@ -32,6 +32,7 @@
 public class TimeoutConfig {
 
   private static final String DEFAULT_TTL_KEY = "default";
+  public static final String STYX_STALE_STATE_TTL_CONFIG = "styx.stale-state-ttls";
```

Comment:
This is a bit strange as it defines a constant but does use it by itself. I don't mind duplicating this in both api and scheduler.

Reply:
yes I thought about this, its also duplicated in the TimeoutConfig test though

Reply:
ups just saw I accidently removed this when rebasing